userguide/cli.rst

```diff
@@ -985,6 +985,17 @@ To collect the output of every module, use :literal:`-A`:
 
    freenas-debug -A
 
+For collecting debug information about a single pool, use
+:command:`zdb` with :literal:`-U /data/zfs/zpool.cache`
+followed by the name of the pool:
```

Comment:
In the old UI, we are still using the term "volume".  The change to this branch should mention both:
"followed by the name of the volume (ZFS pool):"